.circleci/config.yml

```diff
@@ -8,17 +8,16 @@ install_hex_rebar: &install_hex_rebar
       mix local.rebar --force
 
 jobs:
-  build_elixir_1_11_otp_23:
+  build_elixir_1_12_otp_24:
     docker:
-      - image: hexpm/elixir:1.11.2-erlang-23.1.2-alpine-3.12.1
+      - image: hexpm/elixir:1.12.1-erlang-24.0.2-alpine-3.13.3
     steps:
       - checkout
       - <<: *install_hex_rebar
       - restore_cache:
           keys:
             - v1-mix-cache-{{ checksum "mix.lock" }}
       - run: mix deps.get
-      - run: mix format --check-formatted
```

Comment:
Could you enable this line? I.e., use the Elixir 1.12 formatter? It will be a super simple change and it will make the code nicer if it's what I think it is.

Reply:
From what I read (src: https://elixirforum.com/t/did-mix-format-changed-with-elixir-1-12/39836/3), the formatter changed slightly in Elixir 1.12